armando/block-templates/page-sidebar-left.html

```diff
@@ -1,46 +1,25 @@
 <!-- wp:template-part {"slug":"header","tagName":"header","className":"site-header is-style-armando-no-bottom-margin"} /-->
-
 <!-- wp:columns {"align":"full","className":"armando-has-sidebar-left is-style-armando-no-bottom-margin"} -->
 <div class="wp-block-columns alignfull armando-has-sidebar-left is-style-armando-no-bottom-margin">
 	<!-- wp:column {"width":"340px"} --><div class="wp-block-column" style="flex-basis:340px">
-	<!-- wp:template-part {"slug":"sidebar-left","tagName":"aside","backgroundColor":"light-blue","className":"armando-sidebar-left"} /-->
+	<!-- wp:template-part {"slug":"sidebar-left","tagName":"aside","backgroundColor":"silver","className":"armando-sidebar-left"} /-->
```

Comment:
I'm not sure I understand why there are two identical sidebar-* template parts.  Perhaps that could be a single template part used by both templates in different ways...

Or maybe you're achieving something I'm not aware of and that's just what you're going for!

Reply:
Originally, there was a template for displaying both a left and right sidebar. I am undecided on what the best way is, perhaps it is best to not use a template part at all, so that different posts/pages can have different sidebars.